### PR TITLE
(#2989) Require HTTPS to access the site.

### LINKED ISSUE #2989: Enabler: Enforce HTTPS on non-local environments
# Enforce HTTPS on ODEs and ACSF environments.

The acquia-sites.com and acsitefactory.com domain originally didn't have valid TLS certificates. HTTPS should be enforced on these environments.

### PR DESCRIPTION
**Pull request by blairl-nih**
- Remove 403 for non HTTPS access attempt, simply redirect to HTTPS. This will enforce HTTPS in all environments, regardless of domain.
- Update line numbers for file offsets occuring after .htaccess.https.patch.

Close #2989

### CODE REVIEW

File: docroot/.htaccess [hunk 1]
```diff
@@ -115,17 +115,8 @@ AddEncoding gzip svgz
   # uncomment the following line:
   # RewriteBase /
 
-  # Return Status 403 (Forbidden) on non-ODE server access in the
```
Comment (bryanpizzillo) [inline docroot/.htaccess:118]: It seems like removing this would allow people to bypass dev,test,int urls such as www-dev-ac.cancer.gov.

**Review by bryanpizzillo — COMMENTED** [review]
I would like to know the answer to my comment below before requesting changes. This change looks like it will have additional side effects not discussed in this ticket.

File: docroot/.htaccess (hunk 1 above)
Comment (blairlearn) [reply docroot/.htaccess:118]: Yeah, I removed too much.  That does indeed allow the lower tiers to be accessed by their native names.  (But it does force HTTPS! 😁)

**Review by blairlearn — CHANGES_REQUESTED** [review]
Keep the enforcement of cancer.gov host names.

**Pull request merged**